src/main/java/org/jabref/gui/entryeditor/LatexCitationsTabViewModel.java

```diff
@@ -69,6 +74,9 @@ public LatexCitationsTabViewModel(BibDatabaseContext databaseContext,
         this.citationList = FXCollections.observableArrayList();
         this.status = new SimpleObjectProperty<>(Status.IN_PROGRESS);
         this.searchError = new SimpleStringProperty("");
+
+        this.fileUpdateMonitor = Globals.getFileUpdateMonitor();
```

Comment:
Only a comment. I tried to think whether `com.airhacks.afterburner.injection.Injector#instantiateModelOrService(java.lang.Class<T>)` can be used. But it cannot be used, because it needs an empty constructor... Much more work at `LatexCitationsTabViewModel` required.

Background: The FileUpdateMonitor should not be retrieved by globals, but via the injector. One would need in `org.jabref.gui.entryeditor.LatexCitationsTab#LatexCitationsTab` at the first line (where the ViewModel is instanitated) to use `com.airhacks.afterburner.injection.Injector#instantiateModelOrService(java.lang.Class<T>)`.